estate/models/estate_property.py

```diff
@@ -0,0 +1,111 @@
+from odoo import models, fields, api
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "estate.property"
+
+    name = fields.Char(string="Title", required=True)
+    description = fields.Text(string="Description")
+    postcode = fields.Char(string="Postcode")
+    date_availability = fields.Date(
+        string="Availability From",
+        copy=False,
+        default=lambda self: fields.Date.today() + relativedelta(months=3),
+    )
+    expected_price = fields.Float(string="Expected Price", required=True)
+    selling_price = fields.Float(
+        string="Selling Price",
+        readonly=True,
+        copy=False,
+    )
+    best_price = fields.Float(
+        string="Best Price",
+        compute="_compute_best_price",
+        store=True,
+    )
+    property_type_id = fields.Many2one(
+        string="Property Type",
+        comodel_name="estate.property.type",
+    )
+    tag_ids = fields.Many2many(
+        string="Tags",
+        comodel_name="estate.property.tag",
+    )
+    buyer_id = fields.Many2one(string="Buyer", comodel_name="res.partner")
```

Comment:
```suggestion
    buyer_id = fields.Many2one('res.partner', string="Buyer")
```
`fields.Many2one` has the `comodel_name` as mandatory parameter meaning you can save space and keep it readable by placing the co-model first.
Up to you.